### DISCUSSION TITLE
Automate Build and Testing and deployment

### DISCUSSION THREAD

**Discussion by samii-ullah** [GrapesJS/grapesjs#3797] [Q&A]
Hello Everyone, 
i am working on Grapejs application to build website builder. Could you please tell me how to setup continuous deployment locally. 
Like when i run my react app, it will render html pages and show application running on other port locally, 
i will deploy it on cloud cone and map domain too but that will be 2nd milestone, 
i want to see my application running on browser, website builder is generating code of new website but i want my app to handle the rendering of html pages by ownself. 
can anyone help me to make a road map?
Thank you in advance

**Comment by Heapware** [comment]
We are also facing this issue in publishing website, @funkjedi @dsgh @tomichal @timhoogendijk please let me know if anyone can help to resolve this issue.
